### PR TITLE
Add step 'Activate the new Python environment' in DEVELOPMENT.md

### PR DESCRIPTION
**Pull request by lboue**
Add step 'Activate the new Python environment'

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Development is only possible on a (recent) Linux or MacOS machine. Other operati
 
 - Download/clone the repo to your local machine.
 - Set-up the development environment: `scripts/setup.sh`
+- Activate the new Python environment  `source .venv/bin/activate`
```
Comment (marcelveldt) [inline DEVELOPMENT.md:9]: this should be included in the setup script so if that is not the case, we should adjust that

Comment (marcelveldt) [reply DEVELOPMENT.md:9]: Yeah, its already part of the setup script

Comment (marcelveldt) [reply DEVELOPMENT.md:9]: Basically you should ALWAYS run the setup script to prepare your dev environment

**Comment by lboue** [comment]
Indeed, it should have worked. I'll check it out.

**Pull request closed without merging**